index.d.ts

```diff
@@ -42,6 +44,15 @@ export type Options = {
 	readonly signal?: AbortSignal;
 };
 
+export type IterableOptions = BaseOptions & {
+	/**
+	Maximum number of promises returned by `mapper` that have resolved but not yet collected by the consumer of the async iterable. Calls to `mapper` will be limited so that there is never too much backpressure.
+
+	@default concurrency
```

Comment:
```suggestion
	Default: `options.concurrency`
```

`@default` should generally be a proper value.